azurerm/internal/services/healthbot/healthbot_service_resource_test.go

```diff
@@ -0,0 +1,146 @@
+package healthbot_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance/check"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/healthbot/parse"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+type HealthbotResource struct{}
+
+func TestAccHealthbot_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_healthbot_service", "test")
+	r := HealthbotResource{}
+	data.ResourceTest(t, r, []resource.TestStep{
+		{
+			Config: r.basic(data),
+			Check: resource.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func TestAccHealthbot_requiresImport(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_healthbot_service", "test")
+	r := HealthbotResource{}
+	data.ResourceTest(t, r, []resource.TestStep{
+		{
+			Config: r.basic(data),
+			Check: resource.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.RequiresImportErrorStep(r.requiresImport),
+	})
+}
+
+func TestAccHealthbot_complete(t *testing.T) {
```

Comment:
the complete test shjould only have 1 step - the update test should go basiuc -> complete -> basic